ecosystem/web-wallet/src/pages/Gallery.tsx

```diff
@@ -1,188 +1,145 @@
-  description: string,
-  supply: number,
-  uri: string): Promise<Types.HexEncodedBytes> => {
-  const payload: { function: string; arguments: any[]; type: string; type_arguments: any[] } = {
-    type: 'script_function_payload',
-    function: '0x1::Token::create_unlimited_token_script',
-    type_arguments: [],
-    arguments: [
-      Buffer.from(collectionName).toString('hex'),
-      Buffer.from(name).toString('hex'),
-      Buffer.from(description).toString('hex'),
-      true,
-      supply.toString(),
-      Buffer.from(uri).toString('hex')
-    ]
-  }
-  const aptosClient = new AptosClient(NODE_URL)
-  const txnRequest = await aptosClient.generateTransaction(account.address(), payload)
-  const signedTxn = await aptosClient.signTransaction(account, txnRequest)
-  const res = await aptosClient.submitTransaction(signedTxn)
-  await aptosClient.waitForTransaction(res.hash)
-  return Promise.resolve(res.hash)
+type CollectionDict = Record<string, TokenAttributes[]>
+type StorageDict = Record<string, MetadataJson>
+
+interface GetGalleryItemsProps {
+  aptosAccount: AptosAccountState
+}
+
+const secondaryBorderColor = {
+  light: 'gray.200',
+  dark: 'gray.600'
 }
 
-const createCollection = async (
-  account: AptosAccount,
-  name: string,
-  description: string,
-  uri: string
-): Promise<Types.HexEncodedBytes> => {
-  const payload: Types.TransactionPayload = {
-    type: 'script_function_payload',
-    function: '0x1::Token::create_unlimited_collection_script',
-    type_arguments: [],
-    arguments: [
-      Buffer.from(name).toString('hex'),
-      Buffer.from(description).toString('hex'),
-      Buffer.from(uri).toString('hex')
-    ]
+// this is a temporary workaround until we get the indexer working
+const getGalleryItems = async ({
+  aptosAccount
+}: GetGalleryItemsProps) => {
+  if (!aptosAccount) {
+    return undefined
   }
   const aptosClient = new AptosClient(NODE_URL)
-  const txnRequest = await aptosClient.generateTransaction(account.address(), payload)
-  const signedTxn = await aptosClient.signTransaction(account, txnRequest)
-  const res = await aptosClient.submitTransaction(signedTxn)
-  await aptosClient.waitForTransaction(res.hash)
-  return Promise.resolve(res.hash)
-}
+  const hexAddress = aptosAccount?.address().hex()
+  if (hexAddress) {
+    const collectionDict: CollectionDict = {}
+    const storageDict: StorageDict = {}
 
-const tableItem = async (handle: string, keyType: string, valueType: string, key: any): Promise<any> => {
-  const response = await fetch(`${NODE_URL}/tables/${handle}/item`, {
-    method: 'POST',
-    headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify({
-      key_type: keyType,
-      value_type: valueType,
-      key
+    let accountTransactions = await aptosClient.getAccountTransactions(hexAddress)
+    accountTransactions = accountTransactions.filter((txn) => !txn?.vm_status?.includes('Move abort'))
+    accountTransactions.forEach((transaction: any) => {
```

Comment:
If it's not possible to make this work without `any`, add a comment explaining why it's necessary